open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/manage/model/PeriodicLimit.java

```diff
@@ -0,0 +1,173 @@
+package com.wso2.openbanking.accelerator.consent.extensions.manage.model;
+
+import com.wso2.openbanking.accelerator.consent.extensions.common.ConsentExtensionConstants;
+import com.wso2.openbanking.accelerator.consent.extensions.util.PeriodicTypesEnum;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.*;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalAdjusters;
+
+/**
+ * This class represents a periodic limit for a VRP consent.
+ * It includes the period type, amount, period alignment, cyclic expiry time, and cyclic paid amount.
+ */
+public class PeriodicLimit {
+    private final String periodType;
```

Comment:
```suggestion
    
    private final String periodType;
```

Reply:
Fixed in [568ef27](https://github.com/wso2/financial-open-banking/pull/74/commits/568ef2707581a129c8787365abff5aa6ffd74e1f)